windows-agent/internal/proservices/landscape/utils.go

```diff
@@ -242,6 +242,30 @@ func distributeConfig(ctx context.Context, db *database.DistroDB, landscapeConf
 	}
 }
 
+func filterClientSection(landscapeConf string) (string, error) {
```

Comment:
I would add a comment here to explain what we filter when we don’t want to read the function body.